test/tokens/halmos/ERC20TestHalmos.t.sol

```diff
@@ -181,21 +179,20 @@ contract ERC20TestHalmos is Test, SymTest {
 
         uint256 newBalanceFrom = erc20.balanceOf(from);
         uint256 newBalanceTo = erc20.balanceOf(to);
-        uint256 newBalanceOther = erc20.balanceOf(other);
 
         if (from != to) {
-            assert(newBalanceFrom == oldBalanceFrom - amount);
-            assert(newBalanceTo == oldBalanceTo + amount);
-            assert(oldAllowance >= amount);
-            assert(
+            assertEq(newBalanceFrom, oldBalanceFrom - amount);
+            assertEq(newBalanceTo, oldBalanceTo + amount);
+            assertTrue(oldAllowance >= amount);
+            assertTrue(
                 oldAllowance == type(uint256).max ||
                     erc20.allowance(from, caller) == oldAllowance - amount
             );
         } else {
-            assert(newBalanceFrom == oldBalanceFrom);
-            assert(newBalanceTo == oldBalanceTo);
+            assertEq(newBalanceFrom, oldBalanceFrom);
+            assertEq(newBalanceTo, oldBalanceTo);
         }
 
-        assert(newBalanceOther == oldBalanceOther);
+        assertEq(erc20.balanceOf(other), oldBalanceOther);
```

Comment:
fixing fucking stack-too-deep issue - YOU KNOW SOLIDITY I REALLY START HATING YOU!